conf/pfcron.conf.defaults

```diff
@@ -802,3 +922,41 @@ description=Cache MAC/IP couple for Ubiquiti AP
 #
 # The schedule of task
 schedule=@every 12h
+#
+# local
+#
+# Local task
+local=0
+
+[purge_binary_logs]
+#
+# type
+#
+# Task type
+type=purge_binary_logs
+#
+# status
+#
+# If the task is enabled
+status=enabled
+#
+# description
+#
+# The description of task
+description=Controls the cleanup of the binary logs. Files older than the window will be removed.
+#
+# schedule
+#
+# The schedule of task
+schedule=@every 1h
+#
+# window
+#
+# How long to keep a binary log file before deleting it
+window=1h
+#
+# local
+#
+# Local task
+local=1
+
```

Comment:
remove line

Reply:
I am not sure about having the local defined in the config this could mess up some tasks if modified.

Reply:
I'm personally fine having it in the conf as long as its not exposed in the admin.

This could always be handy for advanced configuration and I hardly see someone changing this by mistake in a conf file